src/sidebar/components/test/UserMenu-test.js

```diff
@@ -206,6 +206,20 @@ describe('UserMenu', () => {
         assert.calledOnce(fakeBridge.call);
         assert.calledWith(fakeBridge.call, 'openNotebook', 'mygroup');
       });
+
+      it('opens the notebook and closes itself when `n` is typed', () => {
+        const wrapper = createUserMenu();
+        // Make the menu "open"
+        wrapper.find('Menu').props().onOpenChanged(true);
+        wrapper.setProps({});
```

Comment:
```suggestion
act(() => {
  wrapper.find('Menu').props().onOpenChanged(true);
});
wrapper.update();
```

Calling `setProps({})` works because it re-renders the component "manually". The above pattern with `act` better simulates what happens in the actual application though (`onOpenChanged` callback updates state and schedules a re-render, `act` flushes pending re-renders).

Reply:
Hmmmm. I had tried it with `act` first and it didn't work. I may have missed a step, however, so I'll try again.